packages/@webex/contact-center/src/types.ts

```diff
@@ -0,0 +1,108 @@
+export enum HTTP_METHODS {
+  GET = 'GET',
+  POST = 'POST',
+  PATCH = 'PATCH',
+  PUT = 'PUT',
+  DELETE = 'DELETE',
+}
+
+export type WebexRequestPayload = {
+  method?: HTTP_METHODS;
+  uri?: string;
+  addAuthHeader?: boolean;
+  headers?: {
+    [key: string]: string | null;
+  };
+  body?: object;
+  statusCode?: number;
+  json?: boolean;
+};
+
+type Listener = (e: string, data?: unknown) => void;
+type ListenerOff = (e: string) => void;
+
+export type ServiceHost = {
+  host: string;
+  ttl: number;
+  priority: number;
+  id: string;
+  homeCluster?: boolean;
+};
+
+export interface CCConfig {
+  cc: {
+    allowMultiLogin: boolean;
+    clientType: string;
+    isKeepAliveEnabled: boolean;
+    force: boolean;
+    metrics: {
+      clientName: string;
+      clientType: string;
+    };
+    logging: {
+      enable: boolean;
+      verboseEvents: boolean;
+    };
+  };
+}
+
+export interface WebexSDK {
```

Comment:
We won't have to use it in other packages immediately but will help us improve by contributions from other teams. Can be inside Webex Core package or webex-common package.

If this is not too much change, we shall just keep it there and import here so that we can introduce this object to other packages also slowly and when we do so, this can be expanded

Reply:
@mkesavan13 @Kesari3008 this can be done in a future PR and if we have any other additional dependencies on the @webex/common package.